api/enki.py

```diff
@@ -190,39 +162,56 @@ def count_loans_and_holds(patron, pin):
             )
 
     def request(self, url, method='get', extra_headers={}, data=None,
-                params=None, exception_on_401=False):
-        """Make an HTTP request, acquiring/refreshing a bearer token
-        if necessary.
-        """
+                params=None, retry_on_timeout=True, **kwargs):
+        """Make an HTTP request to the Enki API."""
         headers = dict(extra_headers)
-        if exception_on_401:
```

Comment:
This code was copied-and-pasted from Overdrive code and assumes that requests are being made using an authorization token, which isn't the case.